### PR TITLE
Feature

### PR DESCRIPTION
**Pull request by LottieRugg**
The external links are not linked up yet. 

Thank you🪲

### CODE REVIEW

File: stylesheet.css [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+    line-height: 1.6;
+}
+html{
+    scroll-behavior: smooth;
+}
+/* Nav bar styling */
+
+nav{
+    display: flex;
+    flex-direction: row;
```
Comment (ashleycoles) [inline stylesheet.css]: No need to set a direction of row since it's the default

File: stylesheet.css [hunk 2]
```diff
@@ -0,0 +1,275 @@
...
+}
+p{
+    padding-bottom:10px;
+}
+/* header button styling */
+.header_button{
+    display: inline-block;
+    font-size: 20px;
+    border-radius: 45px;
+    border-style: none;
```
Comment (ashleycoles) [inline stylesheet.css]: You should be able to remove border-style now that you're not using a button tag

File: stylesheet.css [hunk 3]
```diff
@@ -0,0 +1,275 @@
...
+   display: block;
+   margin-left: auto;
+   margin-right: auto;
+}
+.my_work_two{
+    display: grid;
+    grid-template-columns: 60% 40%;
+    padding-top: 100px;
+    background-color: #d1e0ef;
+}
```
Comment (ashleycoles) [inline stylesheet.css]: You could make this a bit DRY-er.

If you had a base class of `.my_work` that gets assigned to both of your work sections, and then a second class of `.my_work_reverse` that does nothing apart from override the columns to 60/40 when it has both `.my_work` and `.my_work_reverse`.

That'll eliminate the need to repeat all these rules

File: stylesheet.css [hunk 4]
```diff
@@ -0,0 +1,275 @@
...
+    height: 300px;
+    width: 300px;
+    display: block;
+    margin: auto;
+}
+p{
+    padding-bottom:10px;
+}
+/* header button styling */
+.header_button{
```
Comment (ashleycoles) [inline stylesheet.css:80]: I think you could make your buttons a bit dryer, if you had a 'base' button class that contains all the rules that are the same for all of your buttons, you could then create some shared class selectors to modify things like the colour and text sizes. Should help reduce some repetition

File: stylesheet.css [hunk 5]
```diff
@@ -0,0 +1,275 @@
...
+    padding: 10px;
+    color: #000000;
+    width: 90%;
+}
+textarea::placeholder{
+    color: #6a6a6a;
+}
+/* Footer bar styling */
+footer{
+    height: 70px;
```
Comment (ashleycoles) [inline stylesheet.css]: Would suggest not setting a height here, and instead using the padding to get the height you want

File: stylesheet.css [hunk 6]
```diff
@@ -0,0 +1,275 @@
...
+    display: block;
+    padding: 10px;
+    margin: 10px;
+    background-color: #d1e0ef;
+    border: none;
+    width: 90%;
+}
+textarea{
+    margin: 0 10px;
+    padding: 1px 2px;
```
Comment (ashleycoles) [inline stylesheet.css]: Doesn't look like this padding does anything

File: stylesheet.css [hunk 7]
```diff
@@ -0,0 +1,275 @@
...
+.external_button:hover{
+    background-color: #fae3e5;
+}
+
+.external_button{
+    padding:18px;
+    background-color: #FFFFFF;
+    border-radius: 45px;
+    margin: 10px 10px 0 0;
+    display:inline-block;
```
Comment (ashleycoles) [inline stylesheet.css]: Make sure all your rules have a space between the : and the value

File: stylesheet.css [hunk 8]
```diff
@@ -0,0 +1,275 @@
...
+    }
+    .contact_form{
+        padding: 10px;
+        width: 80%;
+    }
+}
+@media only screen and (max-width: 617px){
+    nav a{
+        display: grid;
+		grid-template-columns: 100%;
```
Comment (ashleycoles) [inline stylesheet.css]: Check your indent here

**Review by ashleycoles — CHANGES_REQUESTED** [review]
A few changes to keep you busy. Really nice work Lottie!

File: stylesheet.css (hunk 3 above)
Comment (LottieRugg) [reply stylesheet.css]: nice!

File: stylesheet.css (hunk 4 above)
Comment (LottieRugg) [reply stylesheet.css:80]: Ah great thank you, that's worked really well.

File: stylesheet.css (hunk 8 above)
Comment (LottieRugg) [reply stylesheet.css]: I couldn't see this on my screen, let me know if it's still an issue

**Review by ashleycoles — APPROVED** [review]

**Pull request merged**